packages/@lwc/integration-karma/helpers/test-utils.js

```diff
@@ -720,6 +720,42 @@ window.TestUtils = (function (lwc, jasmine, beforeAll) {
         signalValidator.add(signal);
     }
 
+    const connectContext = Symbol('connectContext');
+    const disconnectContext = Symbol('disconnectContext');
+
+    const contextKeys = {
+        connectContext,
+        disconnectContext,
+    };
+
+    lwc.setContextKeys(contextKeys);
+
+    const defineContext = (fromContext) => {
+        const contextDefinition = (initialValue) => {
+            class MockContextSignal {
```

Comment:
Does this have to be defined inline? Looks like you could pass in `initialValue` (which you already do) as well as `fromContext`.